ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java

```diff
@@ -3048,17 +3059,12 @@ public static void validateAcidPartitionLocation(String location, Configuration
    * @param tree AST
    */
   public static TxnType getTxnType(Configuration conf, ASTNode tree) {
-    final ASTSearcher astSearcher = new ASTSearcher();
-
+    int tp = tree.getToken().getType();
```

Comment:
doesn't seem to be used anywhere

Reply:
Oops!! Was using it while debugging something. Missed to remove it. Will do.